### ISSUE TITLE
[Typescript] threshold wrongly typed?

### ISSUE THREAD

**Issue by maeertin** [snewcomer/intersection-observer-admin#41]
Hi there,

Could it be that `threshold` in the `IOptions` interface is wrongly typed?

https://github.com/snewcomer/intersection-observer-admin/blob/b921f0b2d10808b0f863473e48b8bfbb0c8585b3/src/index.ts#L4-L9
The typing above says it should be of type `number` but the IntersectionObserver API allows threshold to be an array of numbers.

https://github.com/snewcomer/intersection-observer-admin/blob/b921f0b2d10808b0f863473e48b8bfbb0c8585b3/src/index.ts#L221-L223
You seem to already handle somehow threshold as an array.

Thanks!

**Comment by snewcomer** [comment]
@maeertin Yep you are correct!  If you feel like submitted a PR, I'm be happy to review it.  Also I'd be willing to do the work as well.

**Comment by maeertin** [comment]
Hi there @snewcomer, I did a tiny PR with the update found here https://github.com/snewcomer/intersection-observer-admin/pull/42. Not sure if anything else is needed for the PR.

Thanks for the quick response :)

**Comment by maeertin** [comment]
Hi @snewcomer, thanks for looking at the tiny PR :) Any plans on publishing to npm?

**Comment by maeertin** [comment]
Or if you know if it's possible to override an already defined key, `threshold`, within the `IOptions` interface?

```ts
declare module 'intersection-observer-admin' {
  interface IOptions {
    threshold?: number | number[]
  }
}
```
Tried several variations of the above with no luck :(
